mobile/src/pages/ImageCreate.tsx

```diff
@@ -1,56 +1,100 @@
 
     return (
-        <View className="flex-1">
-        <SafeAreaView className="flex-1">
-            <View className="android:mt-16 mb-4 ml-2 flex flex-row items-center">
-                <Pressable>
-                    <Icon
-                        size={30}
-                        name="arrow-back-outline"
-                        onPress={() => navigation.navigate("FeedPage")}
-                    />
-                </Pressable>
-                <View className="grow">
-                    <TopNavBar icon="color-wand-outline" className="m-2" value={search} onChangeText={setSearch} />
-                </View>
-            </View>
-            <ScrollView>
-                <View className="flex flex-row flex-wrap justify-evenly">
-                    {imageStyles.map(style =>
-                        <SelectableChip
-                            key={style}
-                            className="m-2 max-w-fit"
-                            label={style}
-                            selected={false}
-                            onSelect={() => { }}
+        <View className="flex-1 bg-white">
+            <SafeAreaView className="flex-1">
+                <View className="android:mt-16 mb-4 ml-2 flex flex-row items-center">
+                    <Pressable>
+                        <Icon
+                            size={35}
+                            name="arrow-back-outline"
+                            onPress={() => navigation.navigate("FeedPage")}
                         />
-                    )}
-                </View>
-                <View className="flex mt-4 items-center">
-                    <Button name="Create" />
+                    </Pressable>
+                    <View className="grow mt-2">
+                        <TopNavBar icon="color-wand-outline" className="m-2" value={prompt} onChangeText={setPrompt} />
+                    </View>
                 </View>
-                <View className="flex-row items-center">
-                    <Text className="m-3 mr-2 text-2xl">Recently created</Text>
-                    <Icon size={25} name="timer-outline"/>
+                <View className="flex m-6 mt-0 justify-evenly">
+                    <MultiSelect
+                        items={imageStyles.map(x => ({ id: x, name: x }))}
+                        uniqueKey="id"
+                        onSelectedItemsChange={setSelectedStyles}
+                        selectedItems={selectedStyles}
+                        selectedItemTextColor="#312e81" 
+                        selectedItemIconColor="#312e81" 
+                        tagBorderColor="#312e81"
+                        submitButtonColor="#312e81" 
+                        tagTextColor="#312e81" 
+                        tagRemoveIconColor="#ed4242"
+                        tagContainerStyle={{ backgroundColor: "white" }}
+                        styleTextTag={{ }} />
+                    <Button name="Create" onPress={handleCreate} className="mt-4"/>
```

Comment:
while a image is being created, disable this button to prevent spam calls

Reply:
The tags and search don't update as well.

Reply:
@fabriUCF wdym by this?

Reply:
working on the issues now

Reply:
Oh so this essentially means, once an image is created the prompt and tags remain the same (hence the "they don't update"). I didn't know if that was intentional so I just commented it. Sorry lol I thought I dm'd Dani about it.

Reply:
Oh good idea, yeah we should fix that